cms/templates/widgets/deprecated-course-key-warning.html

```diff
@@ -0,0 +1,49 @@
+<%page args="course=None" expression_filter="h" />
+<%!
+from datetime import datetime
+from datetime import date
+
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.translation_utils import translate_date
+
+DEFAULT_LANGUAGE = getattr(settings, 'LANGUAGE_CODE', 'en')
+IS_ENABLED = settings.FEATURES.get('DEPRECATE_OLD_COURSE_KEYS_IN_STUDIO', False)
```

Comment:
Nit: I guess if we're going to provide a default, this should fall-back to the same default used in `common.py`.

Reply:
Updated.